Example/BasicExample/TwiliEngageExampleNotificationService/NotificationService.swift

```diff
@@ -6,6 +6,13 @@
 //
 
 import UserNotifications
+import Segment
+
+extension Analytics {
+    static var main = Analytics(configuration: Configuration(writeKey: "<WRITE_KEY>")
```

Comment:
may need a way to get the writekey from the analytics that user created?

Reply:
they can do that- there's no way to access it from the extension bc `analytics` is always `nil` sooo they can add a constants folder or whatever